ui/app/templates/components/form-field.hbs

```diff
@@ -88,6 +88,12 @@
     inputValue=(get model valuePath)
     onChange=(action (action "setAndBroadcast" valuePath))
   }}
+{{else if (eq attr.options.sensitive true)}}
+  {{masked-input
+    isPassword=true
```

Comment:
Is this a prop that `masked-input` uses?

Reply:
whoops, yeah, that's old. removing now!